lib/Conversion/TorchToLinalg/TorchToLinalg.cpp

```diff
@@ -1383,6 +1384,24 @@ static Value createLinalgPayloadCalculationForElementwiseOp(
                                          payloadArgs[0], constZero);
     return b.create<SelectOp>(loc, pred, payloadArgs[0], constZero);
   }
+  if (auto lrelu = dyn_cast<AtenLeakyReluOp>(op)) {
+    if (!lrelu.getType()
+             .cast<ValueTensorType>()
+             .getDtype()
+             .isa<mlir::FloatType>()) {
+      lrelu.emitError("unimplemented: non-floating point dtype");
+      return nullptr;
+    }
+    Type elementType = payloadArgs[0].getType();
+    Value constZero =
+        b.create<arith::ConstantOp>(loc, FloatAttr::get(elementType, 0.0));
+    Value pred = b.create<arith::CmpFOp>(loc, arith::CmpFPredicate::UGT,
+                                         payloadArgs[0], constZero);
+    Value positivePart = b.create<SelectOp>(loc, pred, payloadArgs[0], constZero);
+    Value negativePart = b.create<SelectOp>(loc, pred, constZero, payloadArgs[0]);
+    Value scaledNegativePart = b.create<arith::MulFOp>(loc, negativePart, payloadArgs[1]);
```

Comment:
The second arg is scalar and should be got from operands[1]. The `payloadArgs` are elements of tensor argument only.

Reply:
Thanks.